GithubConnect/GithubConnect.psm1

```diff
@@ -1,3 +1,4 @@
-$Scriptroot = Split-Path $script:MyInvocation.MyCommand.Path
-
-Get-ChildItem $Scriptroot *.ps1 -Recurse | % { Import-Module $_.FullName }
+$Scriptroot = Split-Path $script:MyInvocation.MyCommand.Path
+
+Get-ChildItem $Scriptroot\Public *.ps1 -Recurse | ForEach-Object { Import-Module $_.FullName }
+. $Scriptroot\*.Tests.ps1 #This is only in still as this was how I found the Module running - Advise that we remove this line on v1 release

```

Comment:
Yeah no problem - Previously this file would run the Pester Tests when the Module was imported and I left/added in that line to keep to how it had functioned.

But will remove and commit the change.